src/server/string_family.cc

```diff
@@ -1237,11 +1215,7 @@ void StringFamily::MSet(CmdArgList args, ConnectionContext* cntx) {
   if (auto status = transaction->ScheduleSingleHop(std::move(cb)); status != OpStatus::OK)
     result = status;
 
-  if (*result == OpStatus::OK) {
-    cntx->SendOk();
-  } else {
-    cntx->SendError(*result);
-  }
+  cntx->SendError(*result);
```

Comment:
Are you sure it works well for memcache?

Reply:
Yes, because exactly this code is inside SinkReplyBuilder::SendError(OpStatus)